fv/pre_dnat_test.go

```diff
@@ -122,16 +122,15 @@ var _ = Context("with initialized Felix, etcd datastore, 2 workloads", func() {
 		Context("with pre-DNAT policy to prevent access from outside", func() {
 
 			BeforeEach(func() {
-				policy := api.NewNetworkPolicy()
-				policy.Namespace = "fv"
+				policy := api.NewGlobalNetworkPolicy()
 				policy.Name = "deny-ingress"
 				order := float64(20)
 				policy.Spec.Order = &order
 				policy.Spec.PreDNAT = true
 				policy.Spec.ApplyOnForward = true
-				policy.Spec.IngressRules = []api.Rule{{Action: "deny"}}
+				policy.Spec.IngressRules = []api.Rule{{Action: api.Deny}}
 				policy.Spec.Selector = "has(host-endpoint)"
-				_, err := client.NetworkPolicies().Create(utils.Ctx, policy, utils.NoOptions)
+				_, err := client.GlobalNetworkPolicies().Create(utils.Ctx, policy, utils.NoOptions)
```

Comment:
Need to switch to GlobalNetworkPolicy for these, because a NP can't select host endpoints since they are not a namespaced resource.